app/controllers/application_drafts_controller.rb

```diff
@@ -34,9 +34,10 @@ def remove_question
 
   def update
     draft_params = params.require(:draft)
-    question_params = draft_params.require(:questions_attributes).permit!
+    question_attributes = draft_params.require(:questions_attributes)
+                                      .permit!.to_h
```

Comment:
It does mean we're permitting any and all questions_attributes, so I could sanitize a bit to make sure that only the question attributes we expect get permitted. I just copied what was done before in that case, thinking that a staff member screwing with question attributes was unlikely and couldn't cause a security problem.
As for the "to_h" thing, params is no longer a hash in Rails 5. `update_questions` in the draft model expects a hash, not an ActionController::Parameters object, so somewhere it needs to get turned into a hash--here seemed reasonable, given that I don't want a model method to deal with params ever.